extensions/map-layers/src/mapLayers.ts

```diff
@@ -40,15 +41,26 @@ export class MapLayersUI {
 
     // _uiItemsProvider always created to provide access to localization.
     MapLayersUI._uiItemsProvider = new MapLayersUiItemsProvider(IModelApp.localization);
-    if (registerItemsProvider)
+    if (registerItemsProvider) {
       UiItemsManager.register(MapLayersUI._uiItemsProvider);
-    else
+    } else {
       ConfigurableUiManager.registerControl(MapLayersWidgetControl.id, MapLayersWidgetControl);
```

Comment:
For 3.0 I don't believe you should even call `ConfigurableUiManager.registerControl(MapLayersWidgetControl.id, MapLayersWidgetControl);` users can still use the following without registration.
```
   <Widget id={MapLayersWidgetControl.id} label={MapLayersWidgetControl.label}
           control={MapLayersWidgetControl} iconSpec={MapLayersWidgetControl.iconSpec}
            applicationData={{ hideExternalMapLayers: false, mapTypeOptions: { supportTileUrl: true, 
            supportWmsAuthentication: true }, fetchPublicMapLayerSources: true }} />
```
They just wouldn't be able to use the alternate approach of definining widget using classId as I show below.

```
   <Widget id={MapLayersWidgetControl.id} label={MapLayersWidgetControl.label}
           classId={MapLayersWidgetControl.id} iconSpec={MapLayersWidgetControl.iconSpec}
            applicationData={{ hideExternalMapLayers: false, mapTypeOptions: { supportTileUrl: true, 
            supportWmsAuthentication: true }, fetchPublicMapLayerSources: true }} />
```